### ISSUE TITLE
Connecting styles from the file inline

### ISSUE THREAD

**Issue by i-panov** [yiisoft/yii2#15911]
I need to create an email with styles. These styles are also used elsewhere, and there are quite a few of them, so I would not want to write them inline, but for email this is the only option, because mail clients remove all non-inline styles. Why do not I find anything like `Html::styleFromFileToInline($filename)`? Add this please!

**Comment by SilverFire** [comment]
Would you like to try to implement it and submit a pull request?

**Comment by rob006** [comment]
So you want a shortcut for

```php
Html::tag('style', file_get_contents($cssFilePath));
```
?

**Comment by i-panov** [comment]
@rob006, Yes, something like that I had in mind. Only formatted in the method given the aliases of the path.

**Comment by Renkas** [comment]
@i-panov this would not work well.

Better use some tools that are specialized for this and truly inline the styles. Like: https://github.com/MyIntervals/emogrifier
